### ISSUE TITLE
A simple user guide?

### ISSUE THREAD

**Issue by casouri** [noctuid/targets.el#25]
This package looks cool! However, for a user who just wants an extra set of clever text objects, what should he/she do?

IOW, if I want to have some text objects like the ones in the README, do I need to add them my self (like in the use-package example) or there are some presets?

Thanks.

**Comment by noctuid** [comment]
If you run `(targets-setup t)`, all default text object keybindings will be changed to use targets instead. For information on customizing these keybindings and presets, see all of the configuration section of the readme. I will move this section higher up or link to it at the beginning of the readme.

Text objects defined by other packages can be easily integrated if they follow the guidelines in the readme (e.g. Lispyville is currently compatible and will optionally use targets in the future).

**Comment by casouri** [comment]
Thanks!

**Comment by casouri** [comment]
Oh, I know what you need! A table of contents!

The reademe is pretty long and a TOC would be very helpful for reading.

**Comment by noctuid** [comment]
Yes definitely. I'll close this once the readme is better structured.

**Comment by NightMachinery** [comment]
I think the readme is pretty good now.
